hrm-service/.sequelizerc

```diff
@@ -2,5 +2,5 @@ const path = require('path');
 
 module.exports = {
   'config': path.resolve('dist', 'config', 'config.js'),
-  'models-path': path.resolve('dist', 'models'),
+  'models-path': path.resolve('dist', 'src', 'models'),
```

Comment:
Is this change correct?

Reply:
Why did this need to change exactly?

Reply:
I have issues running the server locally otherwise :/ Might be something on my side?

![Screenshot from 2022-04-22 12-46-20](https://user-images.githubusercontent.com/15805319/164749273-63419e78-e259-4df4-831b-1a2092d66633.png)

Reply:
The change doesn't seem right to me, but if it's working... 🤷